src/components/ConsoleComponent.js

```diff
@@ -24,71 +21,102 @@ const useStyles = makeStyles((theme) => ({
     boxSizing: 'border-box'
   },
   button: {
+    marginRight: '20px',
+    color: theme.palette.common.white,
+    textTransform: 'none'
+  },
+  problemsButton: {
     padding: 0,
     color: theme.palette.common.white,
     textTransform: 'none'
   },
   expandIcon: {
     width: '29px' // Lines up with padding
   },
-  warning: {
-    paddingRight: '5px',
-    paddingLeft: '20px',
-    color: 'khaki'
-  },
-  error: {
-    paddingRight: '5px',
-    paddingLeft: '20px',
-    color: 'red'
-  },
-  success: {
-    paddingRight: '5px',
-    paddingLeft: '20px',
-    color: 'green'
-  },
   pre: {
     margin: '0px'
+  },
+  circle: {
+    'border-radius': '50%',
+    'min-width': '22px',
+    height: '22px',
+    paddingLeft: 1,
+    paddingRight: 1,
+    background: '#21121d',
+    'align-items': 'center',
+    display: 'flex',
+    marginLeft: 4,
+    'justify-content': 'center'
   }
 }));
 
 export default function Console(props) {
   const classes = useStyles();
+  const [problemsView, setProblemsView] = useState(false);
 
   const toggleExpandConsole = () => {
```

Comment:
I think it is best to separate out the expansion button into its own button at this point. The behavior is a bit confusing when the `Console` button and the `^` button are one and the same. For example, let's say I looked at the `Problems` tab, found nothing of concern, and wanted to close the console. You have to double click, see recording:

https://user-images.githubusercontent.com/52747882/134979016-d2553f52-759a-4d9a-b3de-402cc08a0bd5.mov

Reply:
Yeah, this behavior was strange. I thought it made sense for you to be able to open/close the console with either the `Problems` or `Console` buttons, but it makes more sense for each button to do exactly what it says. Commit `4388dc0` separates that little chevron out into its own button so the behavior is more consistent.

Reply:
I should've been more clear in my initial review, so sorry about that, but what do you think about making it so the `Console` and `Problems` still will open the console to that respective tab? But then if the console is already open, re-pressing the button does not close the tab, and you have to use the chevron to close (same behavior as currently when the console is open). Is that confusing, or does that make sense?

Reply:
I actually really like that, I don't think it's confusing at all. Just made the change in commit `ec40bbb`.